Refactor visuel de la navbar

LinUwUxCat opened the pull request:
Jira associé : https://mmaesgi.atlassian.net/browse/EASYF-20

Utilisation de primevue pour les composants de la navbar

![436487271-365ae566-d94e-4d8b-a1d7-4fbb1fe05e10](https://github.com/user-attachments/assets/3a2b162b-22f4-42fe-a533-5acba0201aea)

LinUwUxCat committed:
Init repo

LinUwUxCat committed:
yes

LinUwUxCat committed:
Refactor visuel de la navbar

Utilisation de primevue pour les composants de la navbar

Zastial commented:
Mettre une police par défaut, si possible : Montserrat, sinon faire valider sur Discord

Zastial commented:
Demander la ville quand on arrive sur le site, l'afficher, sinon ne rien montrer

Zastial commented:
Enlever le bouton "Catégories" pour l'instant il ne nous servira pas

Zastial commented:
Pour les boutons : Langue / Favoris / Compte

Bien s'inspirer du figma style Fever :
![image](https://github.com/user-attachments/assets/13333306-3b22-4ae5-a5fa-858a750ebfd2)

Zastial commented:
Ajouter un border-radius sur l'input de recherche

LinUwUxCat committed:
Feat : Changement de la police en Montserrat

Zastial committed:
Merge branch 'main' into M-Navbar-Changes

LinUwUxCat committed:
Feat : Ajout de la détection de la position
Également, ajout de la ville de l'utilisateur (récupérée avec OSM) a la
navbar lorsque la position est détectée.

LinUwUxCat committed:
Tweak : Changements du css de la navbar pour mieux ressembler a la maquette

LinUwUxCat commented:
![image](https://github.com/user-attachments/assets/9dc2b746-594a-4166-bc07-801207dd4ef9)

Ce qu'il reste a faire : 
- Changer le logo
- Déterminer ou le bouton Carte va (est-ce qu'il amène a une nouvelle page ou descend juste, et si il descend, comment on fait sur les autres pages)
- La version mobile

LinUwUxCat committed:
Ajout de Tailwind et d'une version mobile a la navbar
Ajout de tailwind en tant que plugin vite
La navbar a maintenant une version mobile (la version pc s'active en
taille tailwind "md")
Ajustements des options de la navbar

LinUwUxCat committed:
Feat : Ajout d'un menu de séléction de langue (pc only)

La langue utilisée est désormais affichée a coté du bouton pour changer la langue

Commit:
Fix navbar & use tailwind
